packages/airnode-protocol/contracts/rrp/requesters/RrpBeaconServer.sol

```diff
@@ -36,16 +37,8 @@ contract RrpBeaconServer is
         public
         override sponsorToUpdateRequesterToPermissionStatus;
 
-    mapping(bytes32 => Beacon) private templateIdToBeacon;
-    mapping(bytes32 => bytes32) private requestIdToTemplateId;
-
-    /// @dev Reverts if the template with the ID is not created
-    /// @param templateId Template ID
-    modifier onlyIfTemplateExists(bytes32 templateId) {
-        (address airnode, , ) = airnodeRrp.templates(templateId);
-        require(airnode != address(0), "Template does not exist");
-        _;
-    }
```

Comment:
We can't check if a beacon ID is valid anymore

Reply:
don't see any major drawback to this so seems fine